server/sonar-server/src/main/java/org/sonar/server/permission/ProjectId.java

```diff
@@ -25,21 +25,23 @@
 import static java.util.Objects.requireNonNull;
 
 /**
- * Reference to a project by its db id or uuid. Temporarily
- * as long as permissions do not use only uuids.
+ * Reference to a project by its db id or uuid. The field "id" should
+ * be removed as soon as backend is fully based on uuids.
+ *
+ * @see org.sonar.server.permission.ws.WsProjectRef
  */
 @Immutable
-public class ProjectRef {
+public class ProjectId {
```

Comment:
the id is supposed to be a legacy thing and be dropped asap
still, the name of the class is now Project**Id**.
the `ProjectRef` name seems more logical to me (because it is neither UUID nor ID).
but ok